### ISSUE TITLE
Version bumps

### ISSUE THREAD

**Issue by rsokl** [davidmascharka/tbd-nets#13]
I am able to run our demo on PyTorch 1.1, and I am sure we are Python 3.7 compatible. Can we bump our documented versions?

**Comment by davidmascharka** [comment]
Yeah we might as well test under 1.2 and 1.3 while we're at it. I can do that in the next day or two unless you already have

**Comment by rsokl** [comment]
I haven't tested on anything else. I guess the biggest culprit would probably be the sequence-to-sequence model

P.S. It cracks me up that we didn't even make this installable hahaha.

**Comment by BAZINGAC** [comment]
执行python scripts/extract_features.py \ --input_image_dir 
/CLEVR_v1.0/images/train  \ --output_h5_file data/train_features.h5 \  --model_stage 2
的时候，会报python: can't open file 'scripts/extract_features.py': [Errno 2] No such file or directory
这个错
